website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
+page_title: Network Infrastructure Automation Integration Program <sup>Tech Preview</sup>
+sidebar_title: NIA Integration Program <sup>Tech Preview</sup>
+description: Guide to partnership integrations for Consul NIA
+---
+
+# Network Infrastructure Automation Integration Program <sup>Tech Preview</sup>
+
+HashiCorp’s Network Infrastructure Integration (NIA) Program allows partners to build integrations that allow customers to automate dynamic application workflows leveraging network and security infrastructure at runtime. Detailed below is the approach to integrate your networking technology along with the clearly defined steps, links to information sources, and checkpoints.
+
+## Network Infrastructure Automation
+
+Network Infrastructure Automation (NIA) relies on a declarative, workflow and service driven network automation architecture. NIA leverages an agent `Consul-Terraform-Sync` which leverages Terraform as the underlying automation tool and utilizes the Terraform provider ecosystem to drive relevant change to the network infrastructure. This usage of the Terraform provider in conjunction with Consul-Terraform-Tasks allows for HashiCorp Consul to act as the core data source where it is updated with the state of the infrastructure.
```

Comment:
just want to double-check 'Consul-Terraform-Tasks' in this paragraph is intentional. maybe it's how we're naming tasks?

Reply:
Good catch! It should be Consul-Terraform-Sync. Thanks Lorna!